mlflow/utils/databricks_utils.py

```diff
@@ -162,6 +163,20 @@ def get_databricks_host_creds(profile=None):
         config = provider.ProfileConfigProvider(profile).get_config()
     else:
         config = provider.get_config()
+    # if a path is specified, that implies a Databricks tracking URI of the form:
+    # databricks://profile-name/path-specifier
+    if (not config or not config.host) and path:
+        dbutils = _get_dbutils()
+        # Prefix differentiates users and is provided as path information in the URI
+        key_prefix = path
+        if dbutils:
+            host = dbutils.secrets.get(scope=profile, key=key_prefix + "host")
```

Comment:
Should we delimit the prefix with a `_` or `-` here?

Reply:
I thought about this very briefly and had just let it be... but it seems more natural to have a delimiter -- let's do `-`

Reply:
Agreed - will update that